src/objects.js

```diff
@@ -1,41 +1,85 @@
 const createPerson = (name, age) => {
   // your code here
+  return person = {
+    name:name,
+    age:age,
+  };
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  if (object.hasOwnProperty(property) == true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const isOver65 = person => {
   // your code here
+  if (person.age > 65) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
 const getAges = people => {
   // your code here
+  let arrAges = []
+  for (let i = 0; i < people.length; i++) {
+      arrAges.push(people[i].age);
+    };
+  return arrAges;
 };
 
 const findByName = (name, people) => {
   // your code here
+  for (let i = 0; i < people.length; i++) {
+    if (people[i].name == name) {
+      return people[i];
+    };
+  };
 };
 
 const findHondas = cars => {
   // your code here
+  let hondaCars = []
+  for (let i = 0; i < cars.length; i++) {
+    if (cars[i].manufacturer == 'Honda') {
```

Comment:
In this approach use === instead of ==.

And even tho the logic is perfectly working here, and that is the main goal, is a little bit over engineered. 
You could also use the filter method:

`````
return cars.filter(car => car.manufacturer === "Honda")
`````